views/js/qtiCreator/widgets/interactions/helpers/answerState.js

```diff
@@ -134,7 +135,7 @@ define([
                 break;
         }
 
-        if (rp.processingType === 'templateDriven' && !allowCustomTemplate) {
+        if ((rp.processingType === 'templateDriven' && !allowCustomTemplate) || !features.isVisible('taoQtiItem/creator/interaction/response/responseProcessing/custom')) {
```

Comment:
we could probably move it ('taoQtiItem/creator/interaction/response/responseProcessing/custom') to the `..Key` constant also, for better readability